plugins/main/public/components/common/welcome/welcome.scss

```diff
@@ -1,14 +1,3 @@
-.wz-welcome-page .euiCard .euiTitle,
-.wz-module-body .euiCard .euiTitle {
-  font-size: 16px;
```

Comment:
**question**: Is unsetting the font-size styles intentional? I do not see any reference in the issue or pull request.

Reply:
The classes `.wz-welcome-page` and `.wz-module-body` are no longer in use.